modules/rostests/apitests/user32/LoadImageGCC.c

```diff
@@ -0,0 +1,256 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for LoadImageW using DLL compiled with MSVC
+ * PROGRAMMER:  Doug Lyons <douglyons@douglyons.com>
+ * COPYRIGHT:   Copyright 2024 Doug Lyons <douglyons@douglyons.com>
+ * NOTE:        Works for ReactOS, but not in Windows 2003 Server SP2
```

Comment:
```suggestion
 * PROJECT:     ReactOS API tests
 * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
 * PURPOSE:     Test for LoadImageW using DLL compiled with MSVC
 * COPYRIGHT:   Copyright 2024 Doug Lyons <douglyons@douglyons.com>
 *
 * NOTES:
 * Works on ReactOS, but not on Windows 2003 Server SP2.
```

Also, if you prefer, you can move notes to a standalone comment outside of the header.

Reply:
Should be fixed. Thanks.